packages/ui/src/views/create-project-page.tsx

```diff
@@ -0,0 +1,100 @@
+import { useForm } from 'react-hook-form'
+
+import { Button, Card, CardContent, CardHeader, CardTitle, Icon, Input, Label, Spacer, Text } from '@/components'
+import { SandboxLayout, TranslationStore } from '@/views'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { z } from 'zod'
+
+interface PageProps {
+  isLoading?: boolean
+  onFormSubmit: (data: InputProps) => void
+  apiError?: string | null
+  useTranslationStore: () => TranslationStore
+}
+interface InputProps {
+  identifier: string
+  description?: string
+  is_public?: boolean
+  parent_ref?: string
+}
+
+const createProjectSchema = z.object({
+  identifier: z.string().min(4, {
+    message: 'Enter a valid project name with at least 4 characters'
+  })
+})
+
+export function CreateProjectPage({ isLoading, onFormSubmit, apiError, useTranslationStore }: PageProps) {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<InputProps>({
+    resolver: zodResolver(createProjectSchema)
+  })
+
+  const onSubmit = (data: InputProps) => {
+    onFormSubmit(data)
+  }
+
+  const { t } = useTranslationStore()
+
+  return (
+    <SandboxLayout.Main hasLeftPanel className="flex items-center justify-center">
+      <SandboxLayout.Content>
+        <Card variant="plain" width="full">
+          <CardHeader>
+            <CardTitle className="flex flex-col place-items-center">
+              <Icon name="create-workspace" size={112} />
+              <Spacer size={4} />
+              <Text size={6} weight="medium" color="primary">
+                {t('views:createProject.newProject', 'Create your new project')}
+              </Text>
+              <Spacer size={2} />
+              <Text size={2} color="tertiaryBackground">
+                {t('views:createProject.description', 'Orginaze your projects, pipelines and more.')}
```

Comment:
spelling and content
```suggestion
                {t('views:createProject.description', 'Organize your repositories, pipelines and more.')}
```